history/titles/c_dalaran.txt

```diff
@@ -6,4 +6,14 @@
 1.1.1={
 	liege=d_dalaran
 }
-552.1.1={holder = 59000} #Antonidas[0]
\ No newline at end of file
+552.1.1={holder = 59000} #Antonidas[0]
+603.12.13={
+	holder = 59120 # Ansirem Runeweaver
+	liege=d_dalaran
```

Comment:
it already have `liege=d_dalaran` above?

Reply:
Fixed in 5a1521f42c014d52a4abca6b9c56f6ba5a4b1829.